service/src/test/kotlin/app/cash/backfila/service/BackfillRunnerTest.kt

```diff
@@ -192,6 +192,7 @@ class BackfillRunnerTest {
             fakeBackfilaClientServiceClient.runBatchRequests.receive()
           }
         ).isNotNull()
+        delay(2000)
```

Comment:
basically just added delays because we have to wait for the every-second task to run now before progress is saved

Reply:
Does waitForOtherCoroutines do what you need it to do now?

Reply:
nope because we actually have to wait for the 1second to pass.